### PR TITLE
add lowpass filter and gain reduction to disabled remote audio

### PR DESCRIPTION
**Pull request by kfinn**
instead of just disabling it

🎧 🎶 

addresses #6

### CODE REVIEW

**Comment by kfinn** [comment]
@samandmoore does this blow up your webrtc changes? 😬

File: app/javascript/components/RoomPreview.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React, { useState } from "react";
+import { Button, Container, Form, Row } from "react-bootstrap";
+import RoomConfigForm from "./RoomConfigForm";
+
+export default function RoomPreview({ onSubmit }: { onSubmit: () => void }) {
```
Comment (kfinn) [inline app/javascript/components/RoomPreview.tsx:5]: this is annoying, but seems necessary... without it, I often hit a case where chrome would blow up on `new AudioContext()` (within `useAudioContext`) on an error that creating an audio context isn't allowed without a user interaction first

so... 🤷

File: app/javascript/components/VolumeVisualizer.tsx [hunk 2]
```diff
@@ -1,19 +1,54 @@
...
-interface VolumeSample {
-    value: number
-    time: number
-}
+export default function VolumeVisualizer({ stream }: { stream: MediaStream }) {
+    const audioContext = useAudioContext()
+    const analyserRef = useRef<AnalyserNode>(null)
+    const [latestVolumeSample, setLatestVolumeSample] = useState(MIN_VOLUME)
+
+    useEffect(() => {
```
Comment (kfinn) [inline app/javascript/components/VolumeVisualizer.tsx:16]: scooped this out from the config form, where it had been living. I never really loved it being there.

**Comment by samandmoore** [comment]
Looks cool! 

I don't think it impacts the negotiation stuff.

**Comment by kfinn** [comment]
Awesome, then I'll merge!

**Pull request merged**